src/app/bug/bug.component.spec.ts

```diff
@@ -114,11 +122,17 @@ describe('BugComponent', () => {
         })
       });
     })
-    /** 
+    /**
+     * Testing the bugClick() method
+     */
     describe("bugClick()", () => {
 
       it("should navigate to the /bug page with the name of selected bug", () => {
+          let bugName = 'butterfly'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/JohnnyGitH/BugEDex/pullRequest?prid=8891152)
```suggestion
          let bugName = "butterfly"
```